pkg/ctl/token/create.go

```diff
@@ -126,6 +135,8 @@ func create(vc *cmdutils.VerbCmd) {
 			"The expire time for a token. e.g. 1s, 1m, 1h")
 		set.BoolVar(&args.base64Encoded, "base64", false,
 			"The secret key is base64 encoded or not.")
+		set.StringVar(&args.headers, "headers", "",
```

Comment:
There is a method `set.StringToStringVar()` to receive a map value from the command line. We can use that instead of parse the map value by ourselves.

Reply:
Good idea, updated.